tests/public_key/test_get_public_key_info.py

```diff
@@ -0,0 +1,240 @@
+"""
+Provide tests for command line interface's public key commands.
+"""
+import json
+import re
+
+from click.testing import CliRunner
+
+from cli.constants import (
+    ADDRESS_REGEXP,
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    HEADER_SIGNATURE_REGEXP,
+    NODE_27_IN_TESTNET_ADDRESS,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+    PUBLIC_KEY_REGEXP,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+PUBLIC_KEY_ADDRESS_PRESENTED_ON_THE_TEST_NODE = 'a23be17addad8eeb5177a395ea47eb54b4a646f8c570f4a2ecc0b1d2f6241c6845181b'
+
+
+def test_get_public_key_info():
+    """
+    Case: get information about public key address by public key address.
+    Expect: dictionary of public key address information, keys matched regexp checking.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'public-key',
+        'get-info',
+        '--address',
+        PUBLIC_KEY_ADDRESS_PRESENTED_ON_THE_TEST_NODE,
+        '--node-url',
+        NODE_27_IN_TESTNET_ADDRESS,
+    ])
+
+    public_key_info = json.loads(result.output).get('result').get('public_key_info')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
```

Comment:
Remove this blank line.

Reply:
@dmytrostriletskyi, I wanted to separate the different asserts, removed blank line.